autogluon/task/tabular_prediction/predictor.py

```diff
@@ -412,6 +412,10 @@ def transform_features(self, dataset=None, model=None, base_models=None):
             Valid models are listed in this `predictor` by calling `predictor.model_names`.
             If a stacker model S exists with `base_models=M`, then setting `base_models=M` is equivalent to setting `model=S`.
             `model=None` is a requirement when specifying `base_models`.
+        return_original_features : bool, default = True
```

Comment:
`return_original_features` option could be clarified a bit more, eg by precisely stating what sort of DF will be returned when it is true vs when it is false, as well as describing for what purposes one might want it to be True vs False.

Reply:
Will clarify in future PR